.github/workflows/ci.yml

```diff
@@ -0,0 +1,130 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - master
+
+  workflow_dispatch: ~
+
+env:
+  CMAKE_VERSION: 3.18.3
+  NINJA_VERSION: 1.10.1
+  CMAKE_CONFIG_TYPE: Debug
+  CTEST_OUTPUT_ON_FAILURE: 1
+  NINJA_STATUS: '[%f/%t %o/sec] '
+
+jobs:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - os: ubuntu-latest
+            cc: clang-10
+            cxx: clang++-10
+            experimental: true
+
+          - os: macos-latest
+            cc: clang
```

Comment:
Is it OK that ubuntu uses clang-10 and macOS doesn't specify a version?

Reply:
[`macos-latest` comes with Clang 10](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#language-and-runtime) installed, so I don't see a reason to specify a version.